assets/scripts/wplf-form.js

```diff
@@ -26,6 +26,7 @@
 
       fetch(ajax_object.ajax_url  + '?action=wplf_submit', {
         method: "POST",
+        credentials: ajax_object.ajax_credentials || 'omit', // if it's not defined, omit. 
```

Comment:
That's also kinda true. 

But this fixes the problem for new users, and shouldn't break any existing sites. (Not that suddenly including cookies should _break_ anything, but it just might (`$_REQUEST` vs `$_POST`) . 

But I'm open to changing it.